api/src/main/java/com/epam/pipeline/manager/datastorage/providers/aws/s3/S3Helper.java

```diff
@@ -300,6 +303,44 @@ public void restoreFileVersion(String bucket, String path, String version) {
         moveS3Object(client, bucket, new MoveObjectRequest(path, version, path));
     }
 
+    public void restoreFolderVersion(String bucket, String path, RestoreFolderVO restoreFolderVO) {
```

Comment:
Bad name for method: versions for folder does not exist

Reply:
fixed